hugo/lib/createArticle.js

```diff
@@ -15,10 +15,11 @@ module.exports = (entry) => {
     slug,
     description
   } = content;
+  let idxOfPubExMod = body.indexOf('<div class="pubexchange_module"')
```

Comment:
Clever. Maybe just throw in a comment noting what this is all about? I can see myself forgetting what this line is meant for a couple months from now.